ansible/roles/omero-build/tasks/main.yml

```diff
@@ -35,6 +35,13 @@
     - python-pip
     - python-virtualenv
 
+- name: system packages | pip install packages
+  pip:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - genshi
```

Comment:
At the moment the hosts are generally building both, so the need for splitting isn't that great, but certainly can do so if it makes sense.